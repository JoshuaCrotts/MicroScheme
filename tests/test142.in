;
; Setting list variables.
;

(define x '(1 2 3 4))
(set-car! x 30)
x ; (30 2 3 4)
(set-car! (cdr x) 40)
x ; (30 40 3 4)
(set-cdr! (cdr x) '(50 60))
x ; (30 40 50 60)
