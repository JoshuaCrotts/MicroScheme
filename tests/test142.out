(30 2 3 4)
(30 40 3 4)
(30 40 50 60)
